### DISCUSSION TITLE
[Solved] Project version different to API version?

### DISCUSSION THREAD

**Discussion by portlek** [ArcanePlugins/Treasury#85] [Q&A] [answered]
project version is 1.0.0-SNAPSHOT but the api module's version is 1.0-SNAPSHOT.
is it intentional or not?
[version](https://github.com/lokka30/Treasury/blob/master/api/pom.xml#L17)

**Comment by MrNemo64** [comment]
> project version is 1.0.0-SNAPSHOT but the api module's version is 1.0-SNAPSHOT. is it intentional or not? [version](https://github.com/lokka30/Treasury/blob/master/api/pom.xml#L17)

Probably not. As I said [here](https://github.com/lokka30/Treasury/issues/80):
> There are most likely some more inconcistencies in the code/documentation but these are the ones that I had time to find. We should probably do a full scan of the code after the PR are closed to find these mistakes

That one may be just one of a lot that we will find

**Comment by MrIvanPlays** [comment, marked as answer]
We decided that API version shall not include the `patch` revision a.ka it is only `major.minor` instead of `major.minor.patch` according to https://semver.org/ .

So yes it is intentional that the API version is `1.0-SNAPSHOT`. See #46 for more details.
